src/tests/daos_perf.c

```diff
@@ -225,7 +225,6 @@ stride_buf_op(int opc, char *buf, unsigned offset, int size)
 				 * old version will be fully overwritten
 				 */
 				if (ts_single && opc == STRIDE_BUF_SET) {
-					stride_buf.sb_buf[pos] = 0;
```

Comment:
I don't think we can remove this, otherwise daos_perf will report data corruption if overwriting a large single value with a smaller value.

Reply:
Indeed, it was a stab in the dark without really striving to fully grok that code.   I've reverted it now that you pushed the patch.